UInt64.cc

```diff
@@ -272,6 +272,26 @@ UInt64::transform_to_dap2(AttrTable *)
     return NULL;
 }
 
+
+/**
+ * When send_p() is true a description of the instance is added to the inventory and true is returned.
+ * @param inventory
+ * @return True when send_p() is true, false otherwise
+ */
+bool UInt64::is_dap4_projected(std::vector<string> &inventory)
+{
+    bool has_projected_dap4 = false;
+    if(send_p()) {
+        has_projected_dap4 = true;
+        attributes()->has_dap4_types(FQN(), inventory);
+        string entry;
+        entry += type_name() + " " + FQN();
+        inventory.emplace_back(entry);
```

Comment:
See comment about emplace_back in Vector.cc. Short version: 'inventory.emplace_back(type_name() + " " + FQN());'